src/styles/blocks/catalog.scss

```diff
@@ -16,7 +16,14 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
+    cursor: pointer;
     margin-bottom: 48px;
+    transition: $transform-duration ease-in-out 0.5s;
```

Comment:
specify each prop you apply transition for

Reply:
@SanyaBratashchuk done.